<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>854</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background: rgb(24, 93, 255)
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>251</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
background:rgb(5, 32, 130);
	font: 16pt &quot;Castellar&quot;;
}</string>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
background:rgb(5, 32, 130);
	font: 16pt &quot;Castellar&quot;;
}</string>
    </property>
    <property name="text">
     <string>West</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>530</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
background:rgb(5, 32, 130);
	font: 16pt &quot;Castellar&quot;;
}</string>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>610</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
background:rgb(5, 32, 130);
	font: 16pt &quot;Castellar&quot;;
}</string>
    </property>
    <property name="text">
     <string>South</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>771</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
border: 4px solid black;
                                      padding 35px;
                                      background:url(:/! Zork:/images/a.jpg);
                                      background-position: bottom;
                                      background-repeat: no-repeat; 
                                      background-attachment: fixed;
                                      font: 8pt &quot;Castellar&quot;;
                                      color: white;

}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>460</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
background:rgb(5, 32, 130);
	font: 16pt &quot;Castellar&quot;;
}</string>
    </property>
    <property name="text">
     <string>Teleport</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>540</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
background:rgb(5, 32, 130);
	font: 16pt &quot;Castellar&quot;;
}</string>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>750</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
background:rgb(5, 32, 130);
	font: 16pt &quot;Castellar&quot;;
}</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>630</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
background:rgb(5, 32, 130);
	font: 16pt &quot;Castellar&quot;;
}</string>
    </property>
    <property name="text">
     <string>Pick Up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>720</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background:rgb(135, 135, 135);
	font: 8pt &quot;Castellar&quot;;

color: white;
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
background:rgb(135, 135, 135);
	font: 8pt &quot;Castellar&quot;;

color: white;
}</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
background:rgb(255, 170, 0);
	font: 8pt &quot;Castellar&quot;;

}</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
background:rgb(170, 0, 0);
	font: 8pt &quot;Castellar&quot;;

}</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: white;
background:rgb(5, 32, 130);
	font: 16pt &quot;Castellar&quot;;
}</string>
    </property>
    <property name="text">
     <string>Stamina:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: white;
background:rgb(5, 32, 130);
	font: 16pt &quot;Castellar&quot;;
}</string>
    </property>
    <property name="text">
     <string>Health:</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
